cnxpublishing/views/admin.py

```diff
@@ -292,16 +454,22 @@ def get_baking_statuses_sql(get_request):
             'invalid sort: {}'.format(sort))
     if sort == "STATE ASC" or sort == "STATE DESC":
         sort = 'bpsa.created DESC'
-    uuid_filter = get_request.get('uuid', '')
-    author_filter = get_request.get('author', '')
+    uuid_filter = get_request.get('uuid', '').strip()
+    author_filter = get_request.get('author', '').strip()
+    latest_filter = get_request.get('latest', False)
 
     sql_filters = "WHERE"
+    if latest_filter:
+        print latest_filter
```

Comment:
Let's remove this print statement.

Reply:
Ah, yup, a bit of debugging slipped through. :blush: